docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -171,16 +181,16 @@ To use the icon URI attribute in your media controller application, follow these
 1. Set the server icon on the server side:
 
     ```
-    var server = tizen.mediacontroller.createServer();
-    server.updateIconURI("icon.png");
+    var mcServer = tizen.mediacontroller.createServer();
+    mcServer.updateIconURI("icon.png");
     ```
 
 2. Get the server icon URI on the client side:
 
     ```
-    var client = tizen.mediacontroller.getClient();
-    var server_info = client.getLatestServerInfo();
-    console.log(server_info.iconURI);
+    var mcClient = tizen.mediacontroller.getClient();
```

Comment:
Global comment:
Kindly give the language identifier for the code blocks. For information, refer to the MD style guide: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#inline-code-blocks-with-language-identifier

Reply:
Done.